nautobot/extras/views.py

```diff
@@ -1487,6 +1489,34 @@ class JobResultView(generic.ObjectView):
     queryset = JobResult.objects.prefetch_related("job_model", "obj_type", "user")
     template_name = "extras/jobresult.html"
 
+    def queryset_to_csv(self, instance):
```

Comment:
nit: I know this is patterned after `generic.ObjectListView.queryset_to_csv()` but since here this takes an instance rather than a queryset, the name is misleading here. Would something like `instance_to_csv()` be better? Or alternatively should this just be rolled inline under `get()` rather than existing as a separate function at all?

Reply:
I'm a proponent of the first option! I think that's clearer. PR updated to reflect that. Thanks Glenn. I appreciate the review.